### ISSUE TITLE
bodhi is tagging packages into f32 and f32-updates-pending

### ISSUE THREAD

**Issue by nirik** [fedora-infra/bodhi#3680]
See: 

~ koji list-tag-history --all --package xpra | grep 3.0.1-2.fc32
Fri Nov  1 14:59:22 2019: xpra-3.0.1-2.fc32 tagged into f32-updates-candidate by sagitter
Fri Nov  1 14:59:43 2019: xpra-3.0.1-2.fc32 untagged from f32-updates-candidate by autopen
Fri Nov  1 14:59:43 2019: xpra-3.0.1-2.fc32 tagged into f32-updates-testing-pending by autopen
Fri Nov  1 15:00:08 2019: xpra-3.0.1-2.fc32 untagged from f32-updates-testing-pending by bodhi
Fri Nov  1 15:00:09 2019: xpra-3.0.1-2.fc32 tagged into f32 by bodhi [still active]
Fri Nov  1 15:00:09 2019: xpra-3.0.1-2.fc32 tagged into f32-updates-pending by bodhi [still active]

You can see the flow is all expected, until the end. Why is bodhi tagging a rawhide build into f32-updates-pending?

There's quite a pile of packages in there now:

~ koji list-tagged f32-updates-pending | wc -l                  
6426

**Comment by pypingou** [comment]
ome more context:
````
$ koji buildinfo  xpra-3.0.1-2.fc32 
BUILD: xpra-3.0.1-2.fc32 [1407002]
State: COMPLETE
Built by: sagitter
Source: git+https://src.fedoraproject.org/rpms/xpra.git#cc83a5e5148b78bb41d89cfe8563f166600da6e5
Volume: DEFAULT
Task: 38716222 build (rawhide, /rpms/xpra.git:cc83a5e5148b78bb41d89cfe8563f166600da6e5)
Finished: Fri, 01 Nov 2019 21:59:20 CET
Tags: f32 f32-updates-testing-pending f32-updates-pending
````

**Comment by cverna** [comment]
This was solved with bodhi 5.0 deployment (https://koji.fedoraproject.org/koji/buildinfo?buildID=1412117).

We should probably try to untag the old builds that are in f32-updates-candidate
